backend/capellacollab/settings/modelsources/t4c/repositories/models.py

```diff
@@ -44,7 +44,7 @@ class DatabaseT4CRepository(database.Base):
 
 
 class CreateT4CRepository(pydantic.BaseModel):
-    name: str
+    name: str = pydantic.Field(pattern="/^[-a-zA-Z0-9_]+$/")
```

Comment:
```suggestion
    name: str = pydantic.Field(pattern="^[-a-zA-Z0-9_]+$")
```

I assume that this a copy&paste error from the JavaScript regex pattern. `/` in the Python `re`-module matches the character `/`.